portfolio/src/main/webapp/script.js

```diff
@@ -29,7 +29,22 @@ function addRandomQuote() {
 
 
 function getFromDataUsingArrow() {
-  fetch('/data').then(response => response.text()).then((quote) => {
-    document.getElementById('data-container').innerText = quote;
+  fetch('/data').then(response => response.json()).then((data) => {
+
+    const dataListElement = document.getElementById('data-container');
+    dataListElement.innerHTML = '';
+    dataListElement.appendChild(
```

Comment:
This will be modified when convertToJson is changed. Also would be good practice to verify the number of messages.

Reply:
Noted. I know a future part will use user input, so I will definitely keep that in my for variable number of messages.

Reply:
Ack, thanks.